src/Sylius/Bundle/CoreBundle/Resources/config/validation/PromotionRule.xml

```diff
@@ -104,25 +105,27 @@
             </constraint>
 
             <constraint name="Sylius\Bundle\CoreBundle\Validator\Constraints\ChannelCodeCollection">
-                <option name="groups">total_of_items_from_taxon</option>
+                <option name="allowExtraFields">true</option>
+                <option name="groups">sylius_total_of_items_from_taxon</option>
```

Comment:
Wondering if it wouldn't be better to prefix the group with what the subject is smth like: `sylius_promotion_rule_total_of_items_from_taxon`
We might get a naming overlap with other configurations; not really a problem just might be confusing.